src/IssueViz.Security/IssueViz.Security.csproj

```diff
@@ -45,6 +45,9 @@
   <ItemGroup Label="Package references">
     <PackageReference Include="Microsoft.VisualStudio.ComponentModelHost" Version="14.0.25424" />
     <PackageReference Include="Microsoft.VisualStudio.Language.Intellisense" Version="14.0.23205" />
+    <PackageReference Include="Microsoft.VisualStudio.Imaging" Version="15.0.26201" />
```

Comment:
It looks like this is the first time we've used this package in product code. Why is it version 15?
The only existing ref I could find was in test code to v14.

Reply:
My bad, I didn't notice that it was 15. 
I had to import it to reference ImageMoniker in HotspotTableEntry.